flair/models/multitask_model.py

```diff
@@ -260,6 +260,14 @@ def _fetch_model(model_name) -> str:
 
         cache_dir = Path("models")
         if model_name in model_map:
+            if model_name.startswith("hunflair") or model_name == "bioner":
```

Comment:
"hunflair2" also starts with "hunflair", so I think this warning would always be printed.

Reply:
True. However, HunFlair2 will not be loaded as MultitaskModel. I fix it anyway.